lib/SIL/IR/SILFunction.cpp

```diff
@@ -554,9 +553,6 @@ ResilienceExpansion SILFunction::getResilienceExpansion() const {
   // attribute is preserved if the current module is in
   // the same package, thus should be in the same resilience
   // domain.
-  if (isSerializedForPackage() == IsSerializedForPackage)
```

Comment:
Wait I don't understand why these lines were removed.

The resilience expansion inside of a [serialized_for_package] function is Maximal -- we allow loadable types.